pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -28,9 +34,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
-	routev1 "github.com/openshift/api/route/v1"
-
-	apihelpers "github.com/openshift/hive/pkg/apis/helpers"
+	apiHelpers "github.com/openshift/hive/pkg/apis/helpers"
```

Comment:
Keep package aliases all lower case.

Reply:
Is there any technical reasons for this?

Reply:
No technical reason that I am aware of. Just convention.

https://golang.org/doc/effective_go.html#package-names